### PR TITLE
Just make it render

### PR DESCRIPTION
**Pull request by bdougie**
This is a PR just to get the page to render. Turns out you had a [src/routes.js](https://github.com/DrianHillman/spotless-app/blob/connect-auth0/src/routes.js#L2) adding a `<Router />` and then you had a `<Router />` in your [App.js](https://github.com/DrianHillman/spotless-app/blob/connect-auth0/src/App.js#L6). This might be because of my recommendation, but at the end of the day your app had two Routers, the App router being called inside the routes.js router and causing a literal Stack Overflow or "maximum call stack exceeded".

As you can see below, I got your project to render. Now that it does, you can focus on getting Auth0 to work.

screenshot 2017-06-22 23 33 01

### CODE REVIEW

**Comment by bdougie** [comment]
@DrianHillman when you get a chance, can you set this project up on Netlify? You can do easily by clicking this button below.


[Deploy to Netlify](https://app.netlify.com/start/deploy?repository=https://github.com/DrianHillman/spotless-app)

File: src/App.js [hunk 1]
```diff
@@ -2,26 +2,8 @@ import React, { Component } from 'react';
...
-import history from './history.js';
-
-require('dotenv').config();
-
-const auth = new Auth();
-const handleAuthentication = (nextState, replace) => {
-  if (/access_token|id_token|error/.test(nextState.location.hash)) {
-    auth.handleAuthentication();
-  }
-}
```
Comment (bdougie) [inline src/App.js:21]: Duplicated code

File: src/App.js [hunk 2]
```diff
@@ -30,55 +12,46 @@ class App extends Component {
...
 
   logout() {
     this.props.auth.logout();
   }
-  
+
   render() {
-    const { isAuthenticated } = auth;
-    
+    const { isAuthenticated } = this.props.auth;
```
Comment (bdougie) [inline src/App.js:29]: Switch back to props.auth

File: src/App.js [hunk 3]
```diff
@@ -30,55 +12,46 @@ class App extends Component {
...
-          isAuthenticated() && (
-              <a onClick={this.logout.bind(this)}>
-                Log Out
-              </a>
-            )
-        }
-      
-        <ApolloProvider client={this.createClient()}>
-          <Playlists />
-        </ApolloProvider>
```
Comment (bdougie) [inline src/App.js:81]: Removed duplicated code

File: src/index.js [hunk 4]
```diff
@@ -3,7 +3,11 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import './index.css';
 
+import { makeMainRoutes } from './routes';
+
+const routes = makeMainRoutes();
+
 ReactDOM.render(
-  <App />,
+  routes,
```
Comment (bdougie) [inline src/index.js:11]: App is being used in routes, I removed App and added routes

**Review by DrianHillman — APPROVED** [review]
I just reviewed this PR and it looks great! I was able to follow these edits— thanks for cleaning up those redundancies & getting this branch back to baseline.

**Pull request merged**